build.gradle

```
@@ -4,7 +4,7 @@ plugins {
 }
 
 group = 'com.zeshanaslam'
-version = '3.5.9'
+version = '3.6.0'

```

Should be 3.5.10 and not 3.6.0

```suggestion
version = '3.5.10'

```